cmd/av/stack_tidy.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/spf13/cobra"
+)
+
+var stackTidyCmd = &cobra.Command{
+	Use:          "tidy",
+	Short:        "tidy up the branch metadata",
+	Long: strings.TrimSpace(`
+Tidy up the branch metadata by removing the deleted / merged branches.
+
+This command detects which branch is deleted or merged, and re-parent the child branches. This
+operates on only av's internal metadata, and it won't delete the actual Git branches.
+	`),
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+		origBranches, err := meta.ReadAllBranches(repo)
+		if err != nil {
+			return err
+		}
+		branches := make(map[string]*meta.Branch)
+		for name, br := range origBranches {
```

Comment:
I don't think this is strictly necessary. We could just do

```
for name := range branches {
    branches[name].Parent = newParent
}
```

Reply:
If you feel that you'd rather do it this way, that's fine too.

Reply:
Nope. You cannot. See https://stackoverflow.com/questions/42605337/cannot-assign-to-struct-field-in-a-map